.github/workflows/ci_suite.yml

```
@@ -89,6 +91,37 @@ jobs:
           source $HOME/BYOND/byond/bin/byondsetup
           tools/build/build --ci dm -DCIBUILDING -DCITESTING -DALL_MAPS
 
+  test_tgs_docker:
+    if: "!contains(github.event.head_commit.message, '[ci skip]')"
+    name: Test TGS Docker
+    runs-on: ubuntu-20.04
+    services:
+      tgs:
+        image: tgstation/server
+        env:
+          Database__DatabaseType: Sqlite
+          Database__ConnectionString: Data Source=TGS_TGTest.sqlite3;Mode=ReadWriteCreate
+          General__ConfigVersion: 4.1.0
+          General__ApiPort: 5000
```

You declare `TGS_API_PORT` in the `env` at the top of this file but never use it anywhere, is that supposed to go here? If so:

```suggestion
          General__ApiPort: ${{ env.TGS_API_PORT }}
```

That should be how you do it?